elements/pfe-tabs/demo/index.html

```diff
@@ -41,18 +41,19 @@
     <link href="../../pfelement/dist/pfelement.min.css" rel="stylesheet">
   </head>
 
-  <body unresolved>
+  <body unresolved  >
+  <div class="light-background"><div class="inner">
     <h1 id="top" class="example-heading"><code>&lt;pfe-tabs&gt;</code></h1>
 
-    <pfe-card class="toc" pfe-color="lightest">
-      <h3 id="toc">Table of contents</h3>
+    <pfe-card style="margin-bottom: 2em;" class="" pfe-color="lightest">
+      <h3 id=" ">Table of contents</h3>
       <ul id="jump-links"></ul>
       <a href="#top">Scroll to top</a>
     </pfe-card>
 
     <a class="scroll-up" title="Scroll to top" href="#top"><pfe-icon size="md" icon="web-caret-thin-up"></pfe-icon></a>
 
-    <section class="pfe-l-grid pfe-m-gutters pfe-m-all-12-col pfe-m-all-6-col-on-xl">
+    <section class=" ">
```

Comment:
grid removed because now there are padding differences in the panel area based on breakpoint, so we need them to be full width to show this

Reply:
Want to delete the class attribute entirely then?